server/paymentProviders/opencollective/manual.js

```diff
@@ -53,7 +53,7 @@ async function processOrder(order) {
 
   const isFeesOnTop = order.data?.isFeesOnTop || false;
 
-  const paymentProcessorFeeInHostCurrency = 0;
+  const paymentProcessorFeeInHostCurrency = order.data?.paymentProcessorFeeInHostCurrency || 0;
```

Comment:
Added this mainly for the tests, but it made sense to add the ability to set payment processor fees on manual payments (eg. to reflect bank transfer fees)